tcms/testplans/views.py

```diff
@@ -1031,12 +1031,12 @@ def order_cases(self):
             # Because the cases sortkey in database is chaos,
             # Most of them are None.
 
-            if not request.GET.get('case'):
+            if not request.POST.get('case'):
```

Comment:
Same comment as above. `order_cases` is not covered at all (maybe move the class outside the method so it can be tested more easily) and also GET vs. POST.